com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -186,11 +186,9 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
             {
                 #if defined(_MAIN_LIGHT_SHADOWS)
                     float4 shadowCoord = TransformWorldToShadowCoord(posWS.xyz);
-                    unityLight.shadowAttenuation = MainLightRealtimeShadow(shadowCoord);
-                    unityLight.shadowAttenuation = ApplyShadowFade(unityLight.shadowAttenuation, posWS.xyz);
+                    unityLight.shadowAttenuation = MainLightShadow(shadowCoord, posWS.xyz, shadowMask, _MainLightOcclusionProbes);
```

Comment:
Is there a difference between _LightOcclusionProbInfo and _MainLightOcclusionProbes?

Reply:
Where do you set _MainLightOcclusionProbes? I can not find it in https://github.com/Unity-Technologies/Graphics/blob/c6b2a6fa7d598650b4d61c63aef424bcbedb4f05/com.unity.render-pipelines.universal/Runtime/DeferredLights.cs